### PR TITLE
Some minor cleanups for TypeParameterMatcher

### PR DESCRIPTION
**Pull request by terrarier2111**
Motivation:

I saw some code in TypeParameterMatcher which could be cleaned up and so i did.

Modification:

Clean up some code in TypeParameterMatcher.

Result:

A bit cleaner code in TypeParameterMatcher.

### CODE REVIEW

**Review by NiteshKant — APPROVED** [review]
LGTM

File: common/src/main/java/io/netty/util/internal/TypeParameterMatcher.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2013 The Netty Project
+ * Copyright 2021 The Netty Project
```
Comment (hyperxpro) [inline common/src/main/java/io/netty/util/internal/TypeParameterMatcher.java:2]: It should be 2013 as this class was created back in 2013. WDYT @normanmaurer?

Comment (normanmaurer) [reply common/src/main/java/io/netty/util/internal/TypeParameterMatcher.java:2]: Yea please revert @terrarier2111

Comment (terrarier2111) [reply common/src/main/java/io/netty/util/internal/TypeParameterMatcher.java]: I reverted the change.

**Review by normanmaurer — APPROVED** [review]

**Comment by normanmaurer** [comment]
@terrarier2111 thanks a lot

**Pull request merged**